### PR TITLE
DateTimePicker: new widget connecting Calendar and DateTimePickerWheel

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1125
[Problem] DateTimePicker has to change look from wheel to calendar
[Solution]
 - Current DateTimePicker move to DateTimePickerWheel
 - New widget implementation to join together Calendar and DateTimePickerWheel

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: src/js/profile/mobile/widget/DateTimePicker.js [hunk 1]
```diff
@@ -423,150 +227,55 @@
...
-
-				if (parentContainer && parentContainer.classList.contains(classes.HOUR_CONTAINER)) {
-					self._setInputValue("hour", event.detail.value, false);
-				} else if (parentContainer && parentContainer.classList.contains(classes.MINUTE_CONTAINER)) {
-					self._setInputValue("minute", event.detail.value, false);
-				} else if (parentContainer && parentContainer.classList.contains(classes.DATE_CONTAINER)) {
-					self._setDate(new Date(self._year, 0, event.detail.value + 1));
-				}
+			prototype._getFormat = function () {
+				self._datetime.option("format");
```
Comment (HunseopJeong) [inline src/js/profile/mobile/widget/DateTimePicker.js]: Isn't there any return values?

Comment (TomaszLukawskiSam) [reply src/js/profile/mobile/widget/DateTimePicker.js]: Of course you are right.
Done.

File: src/js/profile/mobile/widget/DateTimePicker.js [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+				if (self.options.view === "wheel") {
+					return self._datetime.value();
+				} else if (self.options.view === "calendar") {
+					return self._calendar.value();
+				}
+				return self.options.value;
+			};
+
+			/**
+			* Handle event for date time change od DateTime picker
```
Comment (lslachciak) [inline src/js/profile/mobile/widget/DateTimePicker.js]: "od DateTimepicker" -> you mean "in"?

**Review by lslachciak — COMMENTED** [review]
I think we should rename link in examples from 
Date Time Picker (wheel view) to just
Date Time Picker

Also how about updating ScheduleApp? Shouldn't we use this new picker there?
(we can do this in separate PR)

File: src/js/profile/mobile/widget/DateTimePicker.js (hunk 2 above)
Comment (TomaszLukawskiSam) [reply src/js/profile/mobile/widget/DateTimePicker.js]: Done.

**Review by lslachciak — APPROVED** [review]
please resolve merge confilct

**Pull request merged**